lsp-completion.el

```diff
@@ -213,16 +213,16 @@ Return `nil' when fails to guess prefix."
 (defun lsp-completion--to-internal (items)
   "Convert ITEMS into internal form."
   (--> items
-       (-map (-lambda ((item &as &CompletionItem
-                             :label
-                             :filter-text?
-                             :_emacsStartPoint start-point
-                             :score?))
-               `(:label ,(or filter-text? label)
-                 :item ,item
-                 :start-point ,start-point
-                 :score ,score?))
-             it)))
+    (-map (-lambda ((item &as &CompletionItem
```

Comment:
The indentation change for `-->` is recent with https://github.com/magnars/dash.el/commit/92562909a763d766fd81b817418c2e05d35d50e0